moonstone/filtering/filtering.py

```diff
@@ -0,0 +1,44 @@
+from moonstone.core.module_base import BaseModule
+
+
+class Filtering(BaseModule):
```

Comment:
Since we are going to change this in the future it does not really matter but `Filtering` should probably inherits from the `BaseFiltering` class you have created in `base.py`  module, but we can change this when we deal with reformating this class.

Reply:
We gonna deal with this while refactoring this class